custom/aaa/models.py

```diff
@@ -0,0 +1,272 @@
+"""Contains all models for the AAA Convergence Dashboard.
+
+Conventions followed in this file:
+    * Fields ending in _ranges represent a field's history from forms that are useful for filtering.
+      This is only used for fields that are required to filter for specific ranges (usually a date range).
+    * Fields ending in _history represent a field's entire history from forms.
```

Comment:
Do we know how big these could get? Could be useful to profile.

Reply:
I am not sure and it will be different for every indicator where the history is stored. For example things like weight and height of a child are measured every month, but an eligible couple's history of family planning methods are likely much smaller.

Another thing that makes this harder is that for large values, these are stored in TOAST tables and not the main table (similar to how large strings are stored). 

These history methods are only intended to be used in background aggregation tasks or in single beneficiary view tables, so there should not be any large user facing queries that need to use them.